n3fit/evolven3fit_cpp/varflavors.py

```diff
@@ -0,0 +1,51 @@
+"""
+varflavors.py
+
+When producing a PDF with different maximal flavor number than the nf during the fit, we first run
+evolven3fit_cpp by manually selecting a theory id of the theory correspodning to the desired nf
+evolution.
+
+After running evolven3fit_cpp, but before running posfit, this script should be run to replace the
+``AlphaS_MZ'' and ``MZ'' values in the .info file, with the ``alphas'' and  ``Qref'' valuse from the
+theory database.
```

Comment:
Do we actually still need this file? Is there no way to do everything in a single shot?

Reply:
No we don't need it, we can do everything in a single shot :)